### PR TITLE
TG-115: Verified employeeLogged props

### PR DESCRIPTION
**Pull request by paurinaldi**
Added verifications when using employeeLogged props so the page doesn't break if said employee is deleted.

### CODE REVIEW

File: src/Components/Employee/Projects/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Projects = () => {
 
   const employeeProjects = projects.reduce((acc, project) => {
     const assignedProject = project.members.find((employee) => {
-      return employee?.name?._id === employeeLogged._id;
+      return employee?.name?._id === employeeLogged?._id;
     });
```
Comment (gavafue) [inline src/Components/Employee/Projects/index.js:24]: I don't know if the validation over employee is necessary, I think not

File: src/Components/Employee/Timesheet/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ function EmployeeTimesheets() {
   const employeeLogged = useSelector((state) => state.employees.employeeLogged);
   const timesheets = useSelector((state) =>
     state.timesheets.list
-      .filter((timesheet) => timesheet.employee?._id === employeeLogged._id)
+      .filter((timesheet) => timesheet.employee?._id === employeeLogged?._id)
```
Comment (gavafue) [inline src/Components/Employee/Timesheet/index.js:27]: I don't know if the validation over employee is necessary, I think not

**Review by gavafue — COMMENTED** [review]
I don't know if the validation over employee is necessary, I think not

File: src/Components/Employee/Timesheet/index.js (hunk 2 above)
Comment (paurinaldi) [reply src/Components/Employee/Timesheet/index.js:27]: It is actually necessary, if I deleted the app breaks

**Review by gavafue — APPROVED** [review]

**Review by samueltrilloRR — APPROVED** [review]

**Pull request merged**